doc/source/rllib/rllib-rlmodule.rst

```diff
@@ -138,11 +138,11 @@ RLlib implements the following abstract framework specific base classes:
 
 The minimum requirement is for sub-classes of :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule` is to implement the following methods:
 
-- :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule._forward_train`: Forward pass for training.
+- :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_train`: Forward pass for training.
```

Comment:
Sorry, but I'm still not ok with this.

Users should NOT override the public `forward_...()` methods, because these contain common logic that should not be touched by any subclass. Instead, we ask users - if they want to subclass RLModule - to override the private `_forward_...()` methods, instead. This is not an uncommon thing to do.

Reply:
Ahh, sorry about this - thank you for taking a second look.